style.css

```diff
@@ -0,0 +1,146 @@
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.logo {
+  margin-right: auto;
+  align-items: center;
+  color: #6070ff;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+}
```

Comment:
Nice navbar 📏 but it's not taking `100%` of the screen width which is causing your hamburger to not be on the complete opposite side of your logo. See below 👇🏽 

## Figma
![Screenshot 2023-02-18 at 08 41 09](https://user-images.githubusercontent.com/45739963/219868915-1d368245-c11c-41b2-adc9-fd855e0bfd88.png)

## Your design
![Screenshot 2023-02-18 at 08 41 31](https://user-images.githubusercontent.com/45739963/219868939-4a61385a-e92b-44ae-a28c-6ebdbed4ae51.png)

Also make sure your logo stays on one line as one looks better than two and two messes up the design 😉